divisiondetector/train.py

```diff
@@ -1,7 +1,7 @@
 from argparse import ArgumentParser
 from divisiondetector.datamodules import DivisionDataModule
 from divisiondetector.trainingmodules import DivisionDetectorTrainer
-from divisiondetector.utils import SaveModelOnValidation
+from divisiondetector.utils.utils import SaveModelOnValidation
```

Comment:
Does that work for you? It should only be `from divisiondetector.utils import SaveModelOnValidation`